packages/teams-js/src/public/authentication.ts

```diff
@@ -16,6 +17,13 @@ import { runtime } from './runtime';
  *
  * This object is used for starting or completing authentication flows.
  */
+
+/**
+ * Exceptional APIs telemetry versioning file: v1 and v2 APIs are mixed together in this file
+ */
+const authenticationTelemetryVersionNumber_v1: ApiVersionNumber = ApiVersionNumber.V_1;
+const authenticationTelemetryVersionNumber_v2: ApiVersionNumber = ApiVersionNumber.V_2;
```

Comment:
@ndangudubiyyam Niharika please specifically take a look at this file. If you have questions on how the version is determined, feel free to ping me or leave comments. Thanks.